frontend/src/views/Logout.vue

```diff
@@ -26,8 +28,8 @@ export default {
   },
   methods: {
     deleteCookie(name) {
-      document.cookie = name + "=;expires=Thu, 01 Jan 1970 00:00:01 GMT;";
-    }
+      document.cookie = name + "=;expires=Thu, 01 Jan 1970 00:00:01 GMT;path=/";
```

Comment:
UserKit sets cookies on the root path and they don't get cleared without this specified. 

Is this ok with line 26: `this.deleteCookie("csrf_base_v3");`?